index.html

```diff
@@ -745,129 +758,172 @@ <h2 class="display-4 text-center" id="data_header">Loading data...</h2>
             </li>
           </ul>
           <!-- Mage Frost -->
           <ul class="tc-box" id="tc_mage_frost" hidden>
+            <li><span class="translate_frost">Frost</span> Guides
+              <a href="https://www.icy-veins.com/wow/frost-mage-pve-dps-guide" target="blank">Icy Veins</a>&nbsp;/&nbsp;
+              <a href="https://www.wowhead.com/frost-mage-guide" target="blank">Wowhead</a>
+            </li>
             <li>
               <a href="https://www.altered-time.com/forum/viewforum.php?f=5" target="blank">Altered Time</a>
             </li>
             <li>Discord
-              <a href="https://discord.gg/0gLMHikX2aZ23VdA" target="blank">Hall of the Guardians</a>
+              <a href="https://discord.me/alteredtime" target="blank">Altered Time</a>
             </li>
           </ul>
           <!-- Monk Brewmaster -->
           <ul class="tc-box" id="tc_monk_brewmaster" hidden>
-            <li>
-              <a href="https://www.peakofserenity.com/bfa/brewmaster/" target="blank">Peak of Serenity</a>
+            <li><span class="translate_brewmaster">Brewmaster</span> Guides
+              <a href="https://www.icy-veins.com/wow/brewmaster-monk-pve-tank-guide" target="blank">Icy Veins</a>&nbsp;/&nbsp;
+              <a href="https://www.wowhead.com/brewmaster-monk-guide" target="blank">Wowhead</a>
             </li>
             <li>
-              <a href="https://goo.gl/u7s0nm">Defensive Trinkets</a>
+              <a href="https://www.peakofserenity.com/bfa/brewmaster/" target="blank">Peak of Serenity</a>
             </li>
             <li>Discord
-              <a href="https://discord.gg/0dkfBMAxzTkWj21F" target="blank">PeakOfSerenity</a>
+              <a href="http://discord.gg/peakofserenity" target="blank">PeakOfSerenity</a>
             </li>
           </ul>
           <!-- Monk Windwalker -->
           <ul class="tc-box" id="tc_monk_windwalker" hidden>
+            <li><span class="translate_windwalker">Windwalker</span> Guides
+              <a href="https://www.icy-veins.com/wow/windwalker-monk-pve-dps-guide" target="blank">Icy Veins</a>&nbsp;/&nbsp;
+              <a href="https://www.wowhead.com/windwalker-monk-guide" target="blank">Wowhead</a>
+            </li>
             <li>
               <a href="https://www.peakofserenity.com/bfa/windwalker/" target="blank">Peak of Serenity</a>
             </li>
             <li>Discord
-              <a href="https://discord.gg/0dkfBMAxzTkWj21F" target="blank">PeakOfSerenity</a>
+              <a href="http://discord.gg/peakofserenity" target="blank">PeakOfSerenity</a>
             </li>
           </ul>
           <!-- Paladin Protection -->
           <ul class="tc-box" id="tc_paladin_protection" hidden>
-            <li>
-              <a href="https://www.wowhead.com/protection-paladin-guide" target="blank">Protection Guide</a>
-            </li>
-            <li>
-              <a href="https://goo.gl/u7s0nm">Defensive Trinkets</a>
```

Comment:
New defensive trinket resource: https://www.dreamgrove.gg/guardian/defensive-trinkets/ yes dreamgrove for paladin.